File: app/src/main/java/com/aphoh/muser/base/MuserSoundcloudKeys.java
package com.aphoh.muser.base;

import com.aphoh.muser.BuildConfig;
import com.aphoh.muser.network.SoundcloudKeys;

import org.jetbrains.annotations.NotNull;

/**
 * Created by Will on 9/30/15.
 */
public class MuserSoundcloudKeys implements SoundcloudKeys{
    @NotNull
    @Override
    public String getClientId() {
        return BuildConfig.SC_CLIENT_ID;
    }
}

File: app/src/main/kotlin/com/aphoh/muser/module/MuserSoundcloudKeys.kt
package com.aphoh.muser.module

import com.aphoh.muser.BuildConfig
import com.aphoh.muser.network.SoundcloudKeys

/**
 * Created by Will on 9/30/15.
 */
class MuserSoundcloudKeys : SoundcloudKeys {
    override val clientId: String = BuildConfig.SC_CLIENT_ID
}
